src/server/pfs/server/server_test.go

```diff
@@ -19,113 +19,21 @@ import (
 
 	"github.com/gogo/protobuf/types"
 	pclient "github.com/pachyderm/pachyderm/src/client"
-	"github.com/pachyderm/pachyderm/src/client/auth"
 	"github.com/pachyderm/pachyderm/src/client/pfs"
-	"github.com/pachyderm/pachyderm/src/client/pkg/grpcutil"
 	"github.com/pachyderm/pachyderm/src/client/pkg/require"
-	"github.com/pachyderm/pachyderm/src/client/version"
-	"github.com/pachyderm/pachyderm/src/client/version/versionpb"
-	authtesting "github.com/pachyderm/pachyderm/src/server/auth/testing"
-	"github.com/pachyderm/pachyderm/src/server/pkg/backoff"
 	"github.com/pachyderm/pachyderm/src/server/pkg/hashtree"
 	pfssync "github.com/pachyderm/pachyderm/src/server/pkg/sync"
 	tu "github.com/pachyderm/pachyderm/src/server/pkg/testutil"
 	"github.com/pachyderm/pachyderm/src/server/pkg/uuid"
 
-	etcd "github.com/coreos/etcd/clientv3"
-	"github.com/hashicorp/golang-lru"
 	"golang.org/x/net/context"
 	"golang.org/x/sync/errgroup"
-	"google.golang.org/grpc"
 )
 
 const (
-	testingTreeCacheSize       = 8
-	etcdAddress                = "localhost:32379" // etcd must already be serving at this address
-	localBlockServerCacheBytes = 256 * 1024 * 1024
+	MB = 1024 * 1024
```

Comment:
Linter seems to be fine, I think it's because it's in an _test.go file which means that it doesn't actually get exported since those files are only taken into account during tests.

I don't think it's worth relocating it, the main worry with copied code is that you might change it in one place but not another. But when we're talking about a universal constant like a how many bytes are in a MB there doesn't seem to be much risk of that. And most of the reason I want these as constants is to make later lines shorter. So going from `10 * MB` to `10 * testing.MB`, while not a huge deal would defeat the purpose a bit.